src/localLighthouse.js

```diff
@@ -6,6 +6,20 @@ import options from './lighthouseOptions';
 import writeResults from './helpers/writeResults';
 import { NAME } from './constants';
 
+const lighthouseCaller = async (index, urlsLength, options) => {
+  try {
+    if (options.verbose) {
+      console.log(
+        `${NAME}: Auditing (${index}/${urlsLength}) ${options.url} in Mode:${options.emulatedFormFactor}`
+      );
+    }
+    const lighthouseAuditResult = await localLighthouse(options);
+    return lighthouseAuditResult;
+  } catch (err) {
+    console.log(err);
```

Comment:
We don't want to catch the error here because an error here means something is in fact wrong and it should stop the process. We can also leave the `console.log` where it was (but update the message). And then with those changes we can just delete this function and use `localLighthouse` directly.